src/truststore/_macos.py

```diff
@@ -396,7 +446,7 @@ def _verify_peercerts_impl(
                 status = Security.SecTrustGetTrustResult(
                     trust, ctypes.byref(sec_trust_result_type)
                 )
-                assert status == 0  # TODO: Check status
+                _raise_if_error(status)
```

Comment:
@sethmlarson You can set [`errcheck`](https://docs.python.org/3/library/ctypes.html#ctypes._FuncPtr.errcheck) on the ctypes function to call this automatically so that the caller doesn't need to remember to do it each time it's called. Look at what I did with _handle_win_error in _windows.py for an example.